co-authors-plus.php

```diff
@@ -1071,8 +1071,11 @@ function current_user_can_set_authors( $post = null ) {
 	 * the query_var is changed.
 	 *
 	 * Also, we have to do some hacky WP_Query modification for guest authors
+	 *
+	 * @param string $selection The assembled selection query
```

Comment:
It might be useful to add a @uses http://docs.phpdoc.org/references/phpdoc/tags/uses.html to point to how this modifies the global WP_Query object.

Reply:
You mean something like
`@uses Changes $wp_query->queried_object, $wp_query->queried_object_id, $wp_query->is_author, $wp_query->is_archive, $wp_query->is_404.`
?